src/pages/settings/Profile/PronounsPage.js

```diff
@@ -108,30 +99,21 @@ function PronounsPage(props) {
 
     return (
         <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-            {({safeAreaPaddingBottomStyle}) => (
-                <>
```

Comment:
Same Safe area padding how is this handled now?

Reply:
Answered in the thread above. Fixed, thanks!